src/docs/integrations/vercel.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Vercel Integration"
+---
+
+## Create a Vercel App
+
+To use the Vercel integration you'll need to create your own integration in Vercel. To start, click "Create" in the [Integrations Developer Console](https://vercel.com/dashboard/integrations/console). 
+
+When configuring the app, use the following values:
+
+| Setting                         | Value                                         |
+| ------------------------------- | --------------------------------------------- |
+| Redirect URL                    | `{YOUR_DOMAIN}/extensions/vercel/configure/`  |
+| UI Hook URL                     | `{YOUR_DOMAIN}/extensions/vercel/ui-hook/`    |
+| Delete Hook URL                 | `{YOUR_DOMAIN}/extensions/vercel/delete/`     |
+
+and take note of your client ID, client secret, and integration URL slug (the value you entered in the URL Slug field. This will be used to construct the integration installation URL). Add those to `config.yml` like this:
```

Comment:
```suggestion
Take note of your client ID, client secret, and integration URL slug (the value you entered in the URL Slug field. This will be used to construct the integration installation URL). Add those to `config.yml` like this:
```